cdc/sink/codec/maxwell.go

```diff
@@ -92,30 +94,92 @@ func rowEventToMaxwellMessage(e *model.RowChangedEvent) (*mqMessageKey, *maxwell
 		Type:      model.MqMessageTypeRow,
 	}
 	value := &maxwellMessage{
-		Ts:       e.CommitTs,
+		Ts:       000000000,
 		Database: e.Table.Schema,
 		Table:    e.Table.Table,
 		Data:     make(map[string]interface{}),
 		Old:      make(map[string]interface{}),
 	}
 
+	physicalTime, _ := tsoutil.ParseTS(e.CommitTs)
+	value.Ts = physicalTime.Unix()
 	if e.PreColumns == nil {
 		value.Type = "insert"
 		for _, v := range e.Columns {
-			value.Data[v.Name] = v.Value
+			switch v.Type {
+			case mysql.TypeString, mysql.TypeVarString, mysql.TypeVarchar:
+				if v.Value != nil {
+					str := string(v.Value.([]byte))
+					if v.Flag.IsBinary() {
+						str = strconv.Quote(str)
+						str = str[1 : len(str)-1]
+					}
+					value.Data[v.Name] = str
+				} else {
+					value.Data[v.Name] = nil
+				}
+			default:
+				value.Data[v.Name] = v.Value
+			}
```

Comment:
the logic in those lines is repeated, we can rewrite those lines in to a function.